File: api-client-server/src/main/webapp/components/staff/services/StepService.js
/**
 * Step Service
 * Service for CRUD step to vacancy
 *
 * @author Andrii Blyznuk
 */

app
    .factory('stepService', function ($q, serverService) {

        return {

            createStep: function (step) {
                return serverService.createStep(step);
            }
        }
    });

File: erp-server/src/main/java/com/dengo/erp/service/StepService.java
package com.dengo.erp.service;


import com.dengo.erp.model.Step;
import com.dengo.erp.repository.StepRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

/**
 * Step Service
 * Service Step Repository that binds to and includes methods of obtaining steps
 *
 * @author Andrii Blyznuk
 */

@Service
public class StepService {

    @Autowired
    StepRepository stepRepository;

    public Step save(Step step){
        return stepRepository.save(step);
    }


}
